zebra-state/src/constants.rs

```diff
@@ -19,13 +26,29 @@ pub use zebra_chain::transparent::MIN_TRANSPARENT_COINBASE_MATURITY;
 // TODO: change to HeightDiff
 pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
-/// The database format version, incremented each time the database format changes.
-pub const DATABASE_FORMAT_VERSION: u32 = 25;
+/// The database format major version, incremented each time the on-disk database format has a
+/// breaking change.
+///
+/// Use [`database_format_version_in_code()`] or [`database_format_version_on_disk()`]
+/// to get the full semantic format version.
+pub const DATABASE_FORMAT_VERSION: u64 = 25;
```

Comment:
Do you think this and the new constants are something we should add to the release checkliist ? Something like "check if any change was done to the database and increase  `DATABASE_FORMAT_VERSION`, `DATABASE_FORMAT_MINOR_VERSION` or `DATABASE_FORMAT_PATCH_VERSION` accordingly".

Reply:
Fixed in commit c02f8f4ac0